src/dsmrp1.py

```diff
@@ -25,6 +25,8 @@ def _unit(x):
 
 _tariff = lambda x: 'low' if x == '0002' else ('high' if x == '0001' else x)
 _id = lambda x: x
+_voltage = lambda x: x.split('*')[0]
```

Comment:
I don't know, I'm not a experienced python programmer, I wanted just the raw value not "(223.6*V)", feel free to improve!